modules/datastore/datastore.routing.yml

```diff
@@ -172,7 +172,7 @@ datastore.sql_endpoint.post.api:
     _access: 'TRUE'
 
 datastore.sql_endpoint.settings:
-  path: '/admin/config/dkan/sql_endpoint'
+  path: '/admin/dkan/sql_endpoint'
```

Comment:
We should add a title for this page. It is confusing to just see this: 
![image](https://user-images.githubusercontent.com/2200763/119199829-012c6e00-ba49-11eb-9905-525c9bd809d5.png)